app/src/main/java/at/irfc/app/ui/map/MapScreen.kt

```diff
@@ -1,30 +1,76 @@
 package at.irfc.app.ui.map
 
-import androidx.compose.foundation.background
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.gestures.detectTransformGestures
 import androidx.compose.foundation.layout.*
-import androidx.compose.material3.MaterialTheme
-import androidx.compose.material3.Text
-import androidx.compose.runtime.Composable
-import androidx.compose.ui.Alignment
+import androidx.compose.runtime.*
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.res.stringResource
-import androidx.compose.ui.unit.dp
+import androidx.compose.ui.draw.clipToBounds
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.graphicsLayer
+import androidx.compose.ui.input.pointer.pointerInput
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.layout.onSizeChanged
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.IntSize
 import at.irfc.app.R
 import com.ramcosta.composedestinations.annotation.Destination
 
+const val MIN_SCALE = 1f
+const val MAX_SCALE = 3f
+const val ASPECT_RATIO = 4 / 3f
+
 @Composable
 @Destination
 fun MapScreen() {
     Box(
         modifier = Modifier
-            .fillMaxWidth()
-            .background(MaterialTheme.colorScheme.errorContainer)
-            .padding(8.dp),
-        contentAlignment = Alignment.Center
+            .fillMaxSize()
     ) {
-        Text(
-            text = stringResource(R.string.nav_bar_map),
-            color = MaterialTheme.colorScheme.error
-        )
+        ZoomableImage()
     }
 }
+
+@Composable
+fun ZoomableImage() {
+    var zoom by remember { mutableStateOf(1f) }
+    var offset by remember { mutableStateOf(Offset.Zero) }
+    var size by remember { mutableStateOf(IntSize.Zero) }
+
+    Image(
+        painter = painterResource(id = R.drawable.map),
+        contentDescription = R.string.nav_bar_map.toString(),
+        contentScale = ContentScale.Fit,
+        modifier = Modifier
+            .fillMaxSize()
+            .aspectRatio(ASPECT_RATIO)
```

Comment:
```suggestion
```
I would remove the aspectRatio here, then the image can also take the whole place available and is not cropped to this size when zooming.

Reply:
done